File: HelpSoftware/Help/Help.Web/Web/src/app/EF/DashBoardUser.ts
import { DashBoard, User } from "./EF";

export class DashBoardUser {
  ID_DashBoardUser: Number;
  DashBoard: DashBoard;
  User: User;
  Created: Date;

  deserialize(input) {
    this.ID_DashBoardUser = input.ID_DashBoardUser;
    this.DashBoard = new DashBoard().deserialize(input.DashBoard);
    this.User = new User().deserialize(input.User);
    this.Created = input.Created;
    return this;
  }
}

File: HelpSoftware/Help/Help.EF/Entitys/DashBoardUser.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Help.EF
{
  public class DashBoardUser : IHelpEntity
  {
    [Key]
    public virtual int ID_DashBoardUser { get; set; }

    public virtual DashBoard DashBoard { get; set; }

    public virtual User User { get; set; }

    public virtual DateTime Created { get; set; }

    [Timestamp]
    public virtual byte[ ] TimeStamp { get; set; }

    public string GetEntityName()
    {
      return "DashBoardUser";
    }

    public int GetID()
    {
      return ID_DashBoardUser;
    }

    public bool GetWriteHistoryEntry()
    {
      return true;
    }
  }
}